### ISSUE TITLE
@supports support

### ISSUE THREAD

**Issue by mxstbr** [thysultan/stylis.js#16]
`@supports` does not seem to work correctly. [Reproduction](https://www.webpackbin.com/bins/-KeeZCr0xKfutOfOujxN)

```CSS
// Input:
@supports (display: block) {
  background: mediumseagreen;
}

// Output:
@supports (display: block) {background-color: mediumseagreen;}
```

StackOverflow question that prompted this: https://stackoverflow.com/questions/43562789/using-supports-with-styled-components/43571955#43571955

**Comment by thysultan** [comment]
It's strange to you'd get `background-color: green;` from that i assume `background: mediumseagreen;` might be a typo... Anyway i think i see the problem, it should handle flat css like `@media`, will push a patch for this.

**Comment by mxstbr** [comment]
Sorry that was a typo, fixed the example.

**Comment by thysultan** [comment]
patched in v2.0.4
